sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -238,7 +249,7 @@ def upload_page_blob(
         response = client.create(
             content_length=0,
             blob_content_length=length,
-            blob_sequence_number=None,
+            blob_sequence_number=None,  #type: ignore
```

Comment:
azure\storage\blob\_upload_helpers.py:252: error: Argument "blob_sequence_number" to "create" of "PageBlobOperations" has incompatible type "None"; expected "int"  [arg-type]

How are we passing None here then??

Reply:
Seems like the generated type hint is a bit wrong. Probably because the Swagger is a bit wrong. It has a default value of `0` but `None` is valid and means the request header is just not passed (which is fine). The type ignore is fine here.